lamindb/_record.py

```diff
@@ -32,24 +34,33 @@
 
 
 def serialize(
-    data: Union[Path, str, pd.DataFrame, ad.AnnData], name, format
-) -> Tuple[Any, Path, str, str]:
+    data: Union[Path, CloudPath, str, pd.DataFrame, ad.AnnData], name, format
+) -> Tuple[Any, Union[Path, CloudPath], str, str]:
     """Serialize a data object that's provided as file or in memory."""
     memory_rep = None
-    if isinstance(data, (Path, str)):
-        local_filepath = Path(data)
-        name, suffix = get_name_suffix_from_filepath(local_filepath)
+    if isinstance(data, str):
```

Comment:
Can you add a comment: # convert str to either Path or CloudPath?

Reply:
ok!